frontend/src/index.css

```diff
@@ -0,0 +1,33 @@
+:root {
```

Comment:
Nao entendi esse arquivo, é algum tipo de base css? se for, deve ser utilizado de acordo com o styled components: [GlobalStyle](https://styled-components.com/docs/api#createglobalstyle)